tasks/libs/common/git.py

```diff
@@ -179,7 +180,11 @@ def try_git_command(ctx, git_command):
                     "orange",
                 )
             )
-            do_retry = yes_no_question("Do you want to retry this operation?", color="orange", default=True)
+            do_retry = (
+                False
+                if github_action
+                else yes_no_question("Do you want to retry this operation?", color="orange", default=True)
+            )
```

Comment:
It feels like this could be replaced by checking if a TTY is attached rather that "if running under a CI"